### PR TITLE
Add ENS names resolving / lookup

### PR DESCRIPTION
**Pull request by hewigovens**
This is essential for #311

### CODE REVIEW

File: Trust/EtherClient/ENS/ENSClient.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        )
+        return Promise { seal in
+            Session.send(request) { result in
+                switch result {
+                case .success(let response):
+                    let data = Data(hex: response)
+                    guard data.count == 32 else {
+                        return seal.reject(ENSError.decodeError)
+                    }
+                    let sub = Data(bytes: data.bytes[12...])
```
Comment (vikmeup) [inline Trust/EtherClient/ENS/ENSClient.swift]: what does `12...` does?

Comment (hewigovens) [reply Trust/EtherClient/ENS/ENSClient.swift]: take last 20 (32 - 12 ) bytes for Address, maybe data.suffix(20) is better?

Comment (vikmeup) [reply Trust/EtherClient/ENS/ENSClient.swift]: do you mind adding a comment that this is getting address, otherwise LGTM

Comment (hewigovens) [reply Trust/EtherClient/ENS/ENSClient.swift]: done

File: Trust/Accounts/ViewModels/AccountViewModel.swift [hunk 2]
```diff
@@ -11,17 +11,20 @@ struct AccountViewModel {
     let current: Wallet?
     let walletBalance: Balance?
     let server: RPCServer
+    let ensName: String
```
Comment (vikmeup) [inline Trust/Accounts/ViewModels/AccountViewModel.swift]: can we pas ENS Model instead?

File: Trust/Accounts/Views/AccountViewCell.swift [hunk 3]
```diff
@@ -12,8 +12,8 @@ class AccountViewCell: UITableViewCell {
     @IBOutlet weak var activeView: UIView!
     @IBOutlet weak var glassesImageView: UIImageView!
     @IBOutlet weak var walletTypeImageView: UIImageView!
-    @IBOutlet weak var addressLable: UILabel!
     @IBOutlet weak var balanceLable: UILabel!
+    @IBOutlet weak var addressLable: UILabel!
```
Comment (vikmeup) [inline Trust/Accounts/Views/AccountViewCell.swift]: can you rename `addressLabel` ?

File: Trust/Accounts/ViewModels/AccountViewModel.swift (hunk 2 above)
Comment (hewigovens) [reply Trust/Accounts/ViewModels/AccountViewModel.swift]: we can change this later, a string is enough right now

File: Trust/Accounts/Views/AccountViewCell.swift (hunk 3 above)
Comment (hewigovens) [reply Trust/Accounts/Views/AccountViewCell.swift]: done

**Pull request merged**